ts/components/NewRemindEmailValidationOverlay.tsx

```diff
@@ -65,15 +68,14 @@ const NewRemindEmailValidationOverlay = (props: Props) => {
   const optionEmail = useIOSelector(profileEmailSelector);
   const isEmailValidated = useIOSelector(isProfileEmailValidatedSelector);
   const emailValidation = useIOSelector(emailValidationSelector);
-
   const isFirstOnBoarding = useIOSelector(isProfileFirstOnBoardingSelector);
   const flow = getFlowType(!!isOnboarding, isFirstOnBoarding);
-
+  const [currentTimer, setCurrentTimer] = useState<number>(timerTiming);
```

Comment:
we may avoid using a state to render this information moving to a ref value in order to avoid a complete rerendering of all the screen due to the change of this value, what do you think?

Reply:
I guess that in our case, we need a state that change every second to render the countdown effect. Anyway your suggestion let me think that it's better to put the state inside the Countdown instead inside the page. So that we make only the Counter re-rendering when the state changes.

Reply:
> it's better to put the state inside the Countdown instead of inside the page

Totally agree, I see that this component actually now handles a polling and an update each second (potentially for an infinite time). @Ladirico what do you think?

Reply:
@CrisTofani we used a provider, what do you think?

Reply:
sounds a good idea!